helm/openwhisk/values.yaml

```diff
@@ -335,6 +335,10 @@ grafana:
   - https://raw.githubusercontent.com/apache/openwhisk/master/core/monitoring/user-events/compose/grafana/dashboards/openwhisk_events.json
   - https://raw.githubusercontent.com/apache/openwhisk/master/core/monitoring/user-events/compose/grafana/dashboards/global-metrics.json
   - https://raw.githubusercontent.com/apache/openwhisk/master/core/monitoring/user-events/compose/grafana/dashboards/top-namespaces.json
+  # https://grafana.com/docs/grafana/latest/administration/configuration/#configure-with-environment-variables
+  env:
+  - name: "TEST"
```

Comment:
I think now, it would be good to move the default env to this section in values.yaml.

Reply:
I actually did that when I first created the MR. The only variable I had left was GF_SECURITY_ADMIN_PASSWORD because it already uses another field in the values.yaml file, but one of the tests failed. I'll push that commit again.

Reply:
```system.basic.WskActionTests > Whisk actions should pass parameters bound on creation-time to the action STANDARD_ERROR
    org.scalatest.exceptions.TestFailedException: "2020-06-26T19:27:03.247708024Z stdout: params.param1: test1 2020-06-26T19:27:03.247795526Z stdout:  params.param2: test2" did not include substring "params.payload: this is a test"